light-zk.js/src/account.ts

```diff
@@ -278,6 +288,26 @@ export class Account {
       .digest();
   }
 
+  getUtxoPrefixViewingKey(salt: string): Uint8Array {
+    return this.getDomainSeparatedHashingSecretKey(salt);
+  }
+
+  getDomainSeparatedHashingSecretKey(domain: string): Uint8Array {
+    return blake2b
+      .create(b2params)
+      .update(this.hashingSecret + domain)
+      .digest();
+  }
+
+  generateUtxoPrefixHash(commitmentHash: Uint8Array, length?: number) {
+    const input = Uint8Array.from([
+      ...this.getDomainSeparatedHashingSecretKey("hashing"),
+      ...commitmentHash,
+    ]);
+
+    return blake2b.create(b2params).update(input).digest().slice(0, length);
```

Comment:
instead of slice you could adjust the length in the b2params

Reply:
cool! I didn't know :)

Reply:
replaced local `b2params` with `{dkLen:PREFIX_LENGTH}` 🚀